### PR TITLE
Add quantization tool to python package

### PR DESCRIPTION
**Pull request by yufenglee**
**Description**: Describe your changes.

**Motivation and Context**
- Why is this change required? What problem does it solve?
- If it fixes an open issue, please link to the issue here.

### CODE REVIEW

File: cmake/onnxruntime_python.cmake [hunk 1]
```diff
@@ -204,6 +207,7 @@ add_custom_command(
   COMMAND ${CMAKE_COMMAND} -E make_directory $<TARGET_FILE_DIR:${test_data_target}>/onnxruntime/datasets
   COMMAND ${CMAKE_COMMAND} -E make_directory $<TARGET_FILE_DIR:${test_data_target}>/onnxruntime/tools
   COMMAND ${CMAKE_COMMAND} -E make_directory $<TARGET_FILE_DIR:${test_data_target}>/onnxruntime/tools/featurizer_ops
+  COMMAND ${CMAKE_COMMAND} -E make_directory $<TARGET_FILE_DIR:${test_data_target}>/onnxruntime/quantization
```
Comment (tianleiwu) [inline cmake/onnxruntime_python.cmake:210]: Would it be better to keep original directory structure: /onnxruntime/tools/quantization?

File: cmake/onnxruntime_python.cmake [hunk 2]
```diff
@@ -243,6 +247,9 @@ add_custom_command(
   COMMAND ${CMAKE_COMMAND} -E copy
       ${onnxruntime_python_tools_featurizers_src}
       $<TARGET_FILE_DIR:${test_data_target}>/onnxruntime/tools/featurizer_ops/
+  COMMAND ${CMAKE_COMMAND} -E copy
+      ${onnxruntime_python_quantization_src}
+      $<TARGET_FILE_DIR:${test_data_target}>/onnxruntime/quantization/
```
Comment (tianleiwu) [inline cmake/onnxruntime_python.cmake:252]: Nit: it is better to exclude the test_*.py in the folder.

File: onnxruntime/python/tools/quantization/__init__.py [hunk 3]
```diff
@@ -0,0 +1,2 @@
+from onnxruntime.quantization.quantize import quantize
```
Comment (tianleiwu) [inline onnxruntime/python/tools/quantization/__init__.py:1]: from . import

Comment (yufenglee) [reply onnxruntime/python/tools/quantization/__init__.py:1]: keep consistent with other modules to use absolute path

**Pull request merged**